test/test-handlers.lisp

```diff
@@ -323,7 +323,7 @@ and see what's happening.")
   (let* ((path (get-parameter "path"))
          (file-info (and path
                          (find path *tmp-test-files*
-                               :key 'second :test 'equal))))
+                               :key 'second :test (lambda (a b) (equal a (namestring b)))))))
```

Comment:
Isn't `*tmp-test-files*` a list of triples with elements
1. a pathname of type `pathname`
2. a filename of type `string` (I gather this because above, a `cl-ppcre:replace` is being done on it) (if this is correct, then calling `namestring` isn't right)
3. a content type of type (not sure, irrelevant anyway)

We are keying on `second`, and `path` is a string I believe, so shouldn't we have `'string-equal` as a test?